spec/controllers/link_accounts_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe LinkAccountsController do
+  context "When GOV.UK accounts is not enabled" do
+    describe "GET /email/manage/things-are-changing" do
+      it "returns 404" do
+        get :show
+        expect(response).to have_http_status(:not_found)
+      end
+    end
+  end
+
+  context "When GOV.UK accounts is enabled" do
+    render_views
+
+    around do |example|
+      ClimateControl.modify FEATURE_FLAG_GOVUK_ACCOUNT: "enabled" do
+        example.run
+      end
+    end
+
+    describe "GET /email/manage/things-are-changing" do
+      context "when the page is requested" do
+        before { get :show }
+        let(:html) { Nokogiri.parse(response.body) }
+
+        it "returns 200" do
+          expect(response).to have_http_status(:ok)
+        end
+
+        it "has the expected title" do
+          expect(html.title).to eq("#{I18n.t('link_accounts.things_are_changing.heading')} - GOV.UK")
+        end
+
+        it "has the expected heading" do
+          expect(html.css("h1").text).to eq(I18n.t("link_accounts.things_are_changing.heading"))
+        end
+
+        it "has a button that links to process_govuk_account_path" do
+          expect(html.at(".govuk-button")["href"]).to eq(process_govuk_account_path)
```

Comment:
Wasn't actually sure where this link points to.
Having a snoop around the controller, assume `process_govuk_account_path` is the best one?